src/test/java/task40/JavaScriptAlertTest.java

```diff
@@ -0,0 +1,47 @@
+package task40;
+
+import org.openqa.selenium.*;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class JavaScriptAlertTest {
```

Comment:
If you have page classes for the tests it will be good to have it for alerts and download percentage tests

Reply:
Done